src/main/java/net/minestom/server/coordinate/Pos.java

```diff
@@ -112,6 +113,19 @@ public Pos(@NotNull Point point) {
         return new Pos(x, y, z, yaw, pitch);
     }
 
+    @Contract(pure = true)
+    public @NotNull Pos withLookAt(@NotNull Pos position) {
```

Comment:
The parameter could be a Point instead of a Pos, couldn't it?

Reply:
Indeed, changed this now.